### DISCUSSION TITLE
Go package unfound

### DISCUSSION THREAD

**Discussion by manongerard2002** [kudrykv/latex-yearly-planner#88] [Q&A] [answered]
Hello,

I tried using this repository on my computer, but encounter a problem due to the import of Go packages.

I am currently on a Windows 10 computer and using Ubuntu 18.04.5 with WSL. My go version is go1.10.4 linux/amd64 and my pdflatex version is pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian)

I downloaded the zip of this repository, unzip it and compiled it using `PLANNER_YEAR=2022 PASSES=1 CFG="cfg/base.yaml,cfg/template_breadcrumb.yaml,cfg/sn_a5x.breadcrumb.default.yaml" NAME="sn_a5x.breadcrumb.default" ./single.sh`

My terminal displayed an error : `cmd/plannergen/plannergen.go:10:2: cannot find package "github.com/kudrykv/latex-yearly-planner/app" in any of:
        /usr/lib/go-1.10/src/github.com/kudrykv/latex-yearly-planner/app (from $GOROOT)
        /home/manon/go/src/github.com/kudrykv/latex-yearly-planner/app (from $GOPATH)`

I don't know a lot about the go language so I don't know why I faced this error. What can I do to fix this error?

**Comment by kudrykv** [comment, marked as answer]
Hi!

Go 1.10 is an old version, last release dated February 2018. Use >= 1.20

**Reply by manongerard2002** [reply]
Thanks ! That was indeed the problem.
